homeassistant/components/alexa_devices/binary_sensor.py

```diff
@@ -40,47 +44,47 @@ class AmazonBinarySensorEntityDescription(BinarySensorEntityDescription):
         entity_category=EntityCategory.DIAGNOSTIC,
         is_on_fn=lambda device, _: device.online,
     ),
+    AmazonBinarySensorEntityDescription(
+        key="motion",
+        device_class=BinarySensorDeviceClass.MOTION,
+        is_on_fn=lambda device, key: bool(
+            device.sensors[key].value != SENSOR_STATE_OFF
+        ),
+        is_supported=lambda device, key: device.sensors.get(key) is not None,
+    ),
+)
+
+DEPRECATED_BINARY_SENSORS: Final = (
     AmazonBinarySensorEntityDescription(
         key="bluetooth",
         entity_category=EntityCategory.DIAGNOSTIC,
         translation_key="bluetooth",
-        is_on_fn=lambda device, _: device.bluetooth_state,
+        is_on_fn=lambda device, key: False,
     ),
     AmazonBinarySensorEntityDescription(
         key="babyCryDetectionState",
         translation_key="baby_cry_detection",
-        is_on_fn=lambda device, key: (device.sensors[key].value != SENSOR_STATE_OFF),
-        is_supported=lambda device, key: device.sensors.get(key) is not None,
+        is_on_fn=lambda device, key: False,
     ),
     AmazonBinarySensorEntityDescription(
         key="beepingApplianceDetectionState",
         translation_key="beeping_appliance_detection",
-        is_on_fn=lambda device, key: (device.sensors[key].value != SENSOR_STATE_OFF),
-        is_supported=lambda device, key: device.sensors.get(key) is not None,
+        is_on_fn=lambda device, key: False,
     ),
     AmazonBinarySensorEntityDescription(
         key="coughDetectionState",
         translation_key="cough_detection",
-        is_on_fn=lambda device, key: (device.sensors[key].value != SENSOR_STATE_OFF),
-        is_supported=lambda device, key: device.sensors.get(key) is not None,
+        is_on_fn=lambda device, key: False,
     ),
     AmazonBinarySensorEntityDescription(
         key="dogBarkDetectionState",
         translation_key="dog_bark_detection",
```

Comment:
If we can't fetch data for this anymore, we should remove it, we should not just keep it around for whatever reason

Reply:
There are in the deprecated list that is used to cleanup entities.

Any other suggestion ?

Reply:
I don't we generally delete these entities for people when they stop exposing them